rails_translation_manager.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "rails-i18n"
   spec.add_dependency "activesupport"
 
-  spec.add_development_dependency "bundler", "~> 1.7"
+  spec.add_development_dependency "bundler"
```

Comment:
Could you explain in the commit message why this has been changed? (It looks harmless enough to me though!)